internal/wazeroir/operations.go

```diff
@@ -770,8 +766,8 @@ func (lid LabelID) String() (ret string) {
 	return
 }
 
-func (l LabelID) IsReturnTarget() bool {
-	return l.Kind() == LabelKindReturn
+func (lid LabelID) IsReturnTarget() bool {
```

Comment:
we prefer single character for the receiver so shall we keep `l`?
```suggestion
func (lid LabelID) IsReturnTarget() bool {
```